Automattic/jetpack
# Actions: consolidate into bigger actions with groups

Fixes #16381

#### Changes proposed in this Pull Request:

As we add more actions to our arsenal, the list of things that must run on each PR and each action happening in the repo gets bigger, and harder to understand for new contributors.

Let's try to clean things up a bit:
- Group similar actions into big actions with multiple jobs happening in parallel.
- Consolidate all action-related things into the `.github` directory, organized in clear subdirectories.
- Transform our existing Add Issues action into an action that can handle multiple tasks at once, with only one build step.

**To do**

There are still a number of things we can do (in follow-up PRs in my opinion) to make things better:
- Improve the next milestone detection so it does not automatically assign closed PRs to a milestone that's already past code freeze.
- Keep adding tasks to the gardening action. 
- More that I don't think about right now?

**Screenshots**

![image](https://user-images.githubusercontent.com/426388/86484031-905b6500-bd55-11ea-860b-5753d62621b8.png)

![image](https://user-images.githubusercontent.com/426388/86484046-994c3680-bd55-11ea-8621-31545a264e72.png)


#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

This is not necessarily easy to test, but there are a few things we can look at.

* The changes I mentioned above will be reflected in the list of actions at the bottom of this PR.
* The issue this PR closes should be automatically assigned to me and marked as in progress: #16381
* I've left this PR unassigned to any milestone on purpose. When we merge it, a milestone should be automatically added to the PR.
* Check that the Beta action worked well after this PR's creation
* Check that `master-built` is properly updated after we merge this PR.
* I temporarily uncommented the job that handles our AFK message to test that it was still working. It worked, as you can see in the comments below.

#### Proposed changelog entry for your changes:

* N/A
